.github/workflows/project-codeflare-release.yml

```diff
@@ -38,6 +38,10 @@ on:
         description: 'Owner of target community-operators-prod repository used to open a PR against'
         required: true
         default: 'redhat-openshift-ecosystem'
+      upstream-repository-organization:
```

Comment:
Invalid workflow file: .github/workflows/project-codeflare-release.yml#L1
you may only define up to 10 inputs for a workflow_dispatch event

Reply:
Then can i proceed with option : hard-coding `upstream-repository-organization` parameter ?

Reply:
sure